.github/workflows/integration-tests.yml

```diff
@@ -294,14 +294,23 @@ jobs:
 
       - name: Run python tests on HIP
         run: |
-          cd python
-          pytest --capture=tee-sys -rfs -vvv -n 32 ./test/unit/language/test_core.py \
-              ./test/unit/language/test_annotations.py \
-              ./test/unit/language/test_subprocess.py \
-              ./test/unit/language/test_standard.py \
-              ./test/unit/language/test_compile_errors.py \
-              ./test/unit/language/test_block_pointer.py \
-              ./test/unit/operators/test_flash_attention.py
+          cd python/test/unit
+          pytest --capture=tee-sys -rfs -vvv -n 32 language \
+                 --ignore=language/test_reproducer.py \
+                 --ignore=language/test_conversions.py \
+                 --ignore=language/test_line_info.py
+
+          # Run runtime tests serially to avoid race condition with cache handling
+          python3 -m pytest -vvv runtime
+          # Run hopper tests
+          python3 -m pytest -vs hopper/test_mixed_io.py \
+                                hopper/test_tma_store_gemm.py
+
+      - name: Run partial tests on HIP
+        run: |
+          cd python/test/unit
```

Comment:
Is there a reason why we split these tests into a separate step ?

Reply:
I suppose @scxiao is just following how it is structured on the cuda side, which first ignores operator tests explicitly and then run it separately. Wondering whether there is a reason for that. If not we can merge them on both side. Either way we can do that as a follow up to both side? wdyt?